src/Components/test/testassets/BasicTestApp/Index.razor

```diff
@@ -73,7 +75,7 @@
     @((RenderFragment)RenderSelectedComponent)
 </app>
 
-@code {
+@functions {
```

Comment:
derp ?

Reply:
Undone, I forgot to change it back, the reason is the new test explorer has glitches so I switched to RTM and that doesn't support code, and I wanted my coloring and intellisense. 

As I said, undone.